aries-site/src/layouts/content/Example.js

```diff
@@ -96,6 +138,7 @@ export const Example = ({
   docs,
   figma,
   height,
+  layout,
```

Comment:
Can we name this to be a bit more explicit? 'layout' just seems too generic and applicable too broadly. 

screenContainer? device? formFactor? screenWidth? viewPort?

Reply:
Yes, I like `screenContainer` or maybe `browserContainer`

Reply:
`browserContainer` +1 👍